source/MaterialXContrib/MaterialXNode/CreateMaterialXNodeCmd.cpp

```diff
@@ -54,14 +54,14 @@ void registerFragment(const MaterialXData& materialData, const std::string& ogsX
     const std::string& fragmentName = materialData.getFragmentName();
     if (!fragmentName.empty() && !fragmentString.empty())
     {
-        MString previousOutputDirectory(fragmentManager->getEffectOutputDirectory());
```

Comment:
So, we no longer care about restoring these directories to their previous values? Probably because we need the modified values after this function, e.g. during generation?

Reply:
Yes.